client/views/index.html

```diff
@@ -1,31 +1,26 @@
 <!DOCTYPE html>
 <html ng-app="app">
-
     <head>
-        
         <title>Is It A Word?</title> 
         <meta charset="utf-8" />
         
-        <!-- Neccessary Scripts -->
         <script src="/angular/angular.js"></script>
         <script src="/controllers/app.js"></script>
-        <script src="/controllers/search.js"></script>
+        <script src="/controllers/search.controller.js"></script>
         <script src="/directives/input-form.directive.js"></script>
+        <script src="/services/regex-escape.service.js"></script>
 
         <script src="/jquery/jquery.js"></script>
         <script src="/bootstrap/js/bootstrap.js"></script>
-        <link rel="stylesheet" type="text/css" href="/css/stylesheet.css">
+        <link rel="stylesheet" type="text/css" href="/css/index.css">
         <link rel="stylesheet" type="text/css" href="/bootstrap/css/bootstrap.css">
-   
     </head>
     
     <body>
-        <div class=container ng-controller="SearchController as searchCtrl">
-            
+        <div class=container>
             <h3>Is It A Word?</h3>
-            <!-- Input Form -->
-            <input-form></input-form>
+            
+            <div input-form></div>

```

Comment:
This change is because I turned the input-form directive into an attribute directive.